source/common/grpc/json_transcoder_filter.h

```diff
@@ -74,6 +82,8 @@ class JsonTranscoderConfig : public Logger::Loggable<Logger::Id::config> {
   google::grpc::transcoding::PathMatcherPtr<const Protobuf::MethodDescriptor*> path_matcher_;
   std::unique_ptr<google::grpc::transcoding::TypeHelper> type_helper_;
   Protobuf::util::JsonPrintOptions print_options_;
+
+  bool match_incoming_request_route_{false};
```

Comment:
Nit: You can make this a const if you initialize it in the constructor initializor list.

Reply:
Ping on this one.

Reply:
are you sure i can? this gets set from the `proto_config` passed to the constructor